Source/Core/Core/PowerPC/JitCommon/JitCache.h

```diff
@@ -66,7 +54,7 @@ class ValidBlockBitSet final
 {
 	enum
 	{
-		VALID_BLOCK_MASK_SIZE = 0x20000000 / 32,
+		VALID_BLOCK_MASK_SIZE = 0x100000000 / 32,
```

Comment:
I just hope we're not making the memory usage of our JIT cache too much higher here, what's the overall effect of this patch on that? It seemed pretty wasteful already (I don't fully grok it so I can't judge)